cfgov/jinja2/v1/_includes/molecules/card.html

```diff
@@ -1,4 +1,17 @@
+{# ==========================================================================
+
+   Card
+
+   ==========================================================================
+
+   Description:
+
+   Creates a card with header and body text.
+
+   ========================================================================== #}
+
+{# TODO: This is a prototype and needs to change the text via parameters. #}
 <div class="m-card">
-  <h3 class="h1 m-card_heading">Card Heading</h3>
-  <p class="m-card_body">Card text</p>
+    <h3 class="m-card_heading h1">Card Heading</h3>
```

Comment:
Makes sense, fixed in https://github.com/cfpb/cfgov-refresh/pull/2032/commits/f94aa700e2657cba5e9626c6510cdac5b7edfc1b